src/classes/RD2_OpportunityEvaluationService_TEST.cls

```diff
@@ -1520,6 +1520,68 @@ private class RD2_OpportunityEvaluationService_TEST {
             'Next Year Value should match the expected planned installment and the future open Opp amounts');
     }
 
+    /**
```

Comment:
Let's add test (if not already added) for the AC testing RD with Status and state set to 'Lapsed'.

Reply:
Let's add a test in this class for a Lapsed RD (AC3).